### PR TITLE
Crossdock

### PR DESCRIPTION
**Pull request by isaachier**

### CODE REVIEW

File: crossdock/Server.cpp [hunk 1]
```diff
@@ -0,0 +1,781 @@
...
+
+#include "Server.h"
+
+#include <atomic>
+#include <cstdlib>
+#include <future>
+#include <sstream>
+#include <thread>
+
+#include <nlohmann/json.hpp>
```
Comment (black-adder) [inline crossdock/Server.cpp]: my mind is blown that cpp doesn't provide some generic way to turn json into a class and vice versa.

**Comment by isaachier** [comment]
https://github.com/jaegertracing/cpp-client/issues/15

File: crossdock/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            - go
+            - cpp
+        environment:
+            - WAIT_FOR=test_driver,go,cpp
+            - WAIT_FOR_TIMEOUT=60s
+
+            - CALL_TIMEOUT=60s
+
+            - AXIS_CLIENT=go
+            - AXIS_S1NAME=go,cpp
```
Comment (black-adder) [inline crossdock/docker-compose.yml]: tchannel not ready yet?

Comment (black-adder) [reply crossdock/docker-compose.yml]: ooops meant for line 20

**Pull request merged**